etc/setup/templates/configs/tyk/confs/authMiddleware.js

```diff
@@ -0,0 +1,183 @@
+// ---- Authentication middleware -----
```

Comment:
this should be with lib/tyk

Reply:
will consider this in the next dev branch